### DISCUSSION TITLE
Some APK at 2.9.3 version is ok but 2.10.0 error occurred

### DISCUSSION THREAD

**Discussion by painhell** [iBotPeaches/Apktool#3748] [General]
2.9.3Can handle correctly but 2.10.0 error occurred.
APK File Download Url:https://ucdl.25pp.com/fs08/2024/10/25/2/110_81b2915f906cc156e3cfc6cc99d71049.apk
![screen](https://github.com/user-attachments/assets/6736ed8e-0988-4938-ae0a-c33715a86639)

**Comment by iBotPeaches** [comment]
You can use `-j` to control number of threads. Turn it to 1 if you don't have the memory to thread operations.

**Reply by painhell** [reply]
At 2.10.0 version，When the APK is large and the number of threads is large, decompilation will cause OutOfMemory exceptions, build will never finish and take up a lot of CPU

**Comment by painhell** [comment]
@iBotPeaches 
Find new PROBLEMS ！！！！
This time an error occurred while building with version 2.10.0, but version 2.9.3 was able to complete.
APK File Download URL:https://file-history.liqucn.com/327045fb2125ce05c9d83bcfff66f08f/6777ce55/upload/2017/313/d/com.jingdian.tianxiameishi.android_6.3.4_liqucn.com.apk
![2 10 0](https://github.com/user-attachments/assets/6c7e3da4-2f32-40af-86bf-daf034cc6429)
![2 9 3](https://github.com/user-attachments/assets/98402cf1-8dbc-4203-b638-651d51282ffb)
